BUILD.gn

```diff
@@ -231,6 +231,13 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
         ]
       }
     }
+
+    if (getenv("JAVA_PATH") != "") {
```

Comment:
should use if (build_java_matter_controller) explicitly instead of depending on an environment variable

Reply:
removed, thanks